eox_hooks/tests/test_actions_handler.py

```diff
@@ -0,0 +1,88 @@
+"""This file contains all the test for the actions_handler.py file.
+
+Classes:
+    SingalsTestCase: TestActionHandler.
+"""
+from django.test import TestCase
+from mock import patch
+
+from eox_hooks.actions_handler import action_handler
+
+
+class TestActionHandler(TestCase):
```

Comment:
You always test the default case, you could do the following:

1) Create two classes, one for action_handler  and another one for action_lookup
2) (action_handler class) 

       -  Mock action_lookup
       -  test when the value returned by  action_lookup doesn't raise an exception
       -  test when the value returned by  action_lookup raise an exception and silently is True
       -  test when the value returned by  action_lookup raise an exception and silently is False

3) (action_lookup class) 

       -  Mock import_module
       -  test happy path( the module exist and has the attribute)verify the returned value
       -  test when the module exist and doesn't have the attribute, verify the log, and the returned value this should be custom_action_mock
       -  test when import module raises Import error, verify the log and the returned value, this should be custom_action_mock

Reply:
Nice suggestion, thank you!

I tried to follow them, the main change I did was using silently because the function that must fail is the one running!